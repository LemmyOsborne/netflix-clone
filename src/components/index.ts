export { Jumbotron } from "./Jumbotron/Jumbotron";
export { Footer } from "./Footer/Footer";