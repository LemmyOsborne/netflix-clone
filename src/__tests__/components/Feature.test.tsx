import React from "react";
import { render } from "@testing-library/react";
import { Feature } from "../../components";

describe("<Feature/>", () => {
    it("renders <Feature/>", () => {
        const { container, getByText } = render(
            <Feature>
                <Feature.Title>Unlimited movies, TV shows, and more.</Feature.Title>
                <Feature.SubTitle>Watch anywhere. Cancel anytime.</Feature.SubTitle>
            </Feature>
        )

        expect(getByText("Unlimited movies, TV shows, and more.")).toBeTruthy()
        expect(getByText("Watch anywhere. Cancel anytime.")).toBeTruthy()
        expect(container.firstChild).toMatchSnapshot()
    })
})