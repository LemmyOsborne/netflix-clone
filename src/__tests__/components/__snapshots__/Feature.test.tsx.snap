// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feature/> renders <Feature/> 1`] = `
<div
  class="sc-cidDSM hukYMt"
>
  <h1
    class="sc-jcFjpl eJJHhx"
  >
    Unlimited movies, TV shows, and more.
  </h1>
  <h2
    class="sc-caiLqq iwnfdr"
  >
    Watch anywhere. Cancel anytime.
  </h2>
</div>
`;
